src/main/java/com/scottlogic/filters/KeywordPostFilter.java

```diff
@@ -5,14 +5,20 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Locale;
 import java.util.Objects;
 
-public class KeywordContentPostFilter implements PostFilter {
+public class KeywordPostFilter implements PostFilter {
     private final String Keyword;
 
-    public KeywordContentPostFilter(String keywordInput) {
-        Keyword = keywordInput;
+    public String getKeyword() {
+        return Keyword;
```

Comment:
The `getter` isn't necessarily a bad idea however I don't think you ever need any for these combined filters and sorters and so it indicates that there might be something that could be improved